lib/fluent/plugin/in_tail.rb

```diff
@@ -608,10 +657,12 @@ def on_timer
     end
 
     class TailWatcher
-      def initialize(path, pe, log, read_from_head, update_watcher, line_buffer_timer_flusher, io_handler_build)
+      def initialize(path, ino, pe, log, read_from_head, follow_inodes, update_watcher, line_buffer_timer_flusher, io_handler_build)
```

Comment:
`path` and `ino` can be merged into `PathInodeTuple`?
It also reduces method argument in other places, e.g `unwatch`, `setup_watcher` and others.

Of course, separate PR is ok for me.

Reply:
Oh, that's reasonable. Will do.

Reply:
Fixed in https://github.com/fluent/fluentd/pull/3182/commits/08ee2da9c3ac895b747dbabd2ef2a0cada1bcc25.

Reply:
Ah, wait. Merging path and inode arguments makes slight a bit unstable for testing...
Perhaps we should do in the separated PR.